apps/swifty-bank/app/agreement/detail/page.tsx

```diff
@@ -0,0 +1,63 @@
+import { Heading } from "@swifty/ui";
+import {
+  content,
+  header,
+  list,
+  container,
+  listContent,
+  listDetail,
+} from "./page.css";
```

Comment:
styles라는 객체로 묶어서 내보내주게 되면
여기서 나온 header, list 등 generic한 단어들을 사용할 때, 재선언의 위험이 줄어들게 됩니다!
```suggestion
import styles from "./page.css";
```
이렇게 `import` 하고

```tsx
<li className={styles.list}>리스트</li>
```
위와 같이 쓸 수 있겠죠!
추후 한 페이지에 많은 스타일이 포함될 때 유용한 방식이라고 생각합니다